ingestion/src/metadata/ingestion/source/messaging/redpanda/connection.py

```diff
@@ -34,8 +40,20 @@ def get_connection(connection: RedpandaConnection) -> KafkaClient:
     return get_kafka_connection(connection)
 
 
-def test_connection(client: KafkaClient, _) -> None:
+def test_connection(
+    metadata: OpenMetadata,
+    client: KafkaClient,
+    service_connection: KafkaClient,
+    automation_workflow: Optional[AutomationWorkflow] = None,
+) -> None:
     """
-    Test connection
+    Test connection. This can be executed either as part
+    of a metadata workflow or during an Automation Workflow
     """
-    test_kafka_connection(client, _)
+
+    test_kafka_connection(
+        metadata=metadata,
+        client=client,
+        service_fqn=service_connection,
+        automation_workflow=automation_workflow,
+    )
```

Comment:
Why this is called `test_kafka_connection` and kafka connection is different to this implementation? `service_connection` is `KafkaClient` instead of `RedpandaConnection`, is that correct?

Reply:
if you see line 22 we have imported `kafka.test_connection` and renamed it to `test_kafka_connection` ,

I'll correct the service connection type here!